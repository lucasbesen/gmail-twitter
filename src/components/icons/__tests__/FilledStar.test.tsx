import React from 'react';
import renderer from 'react-test-renderer';

import { FilledStar } from '../';

it('should render FilledStar icon without error', () => {
  const component = renderer.create(<FilledStar />);
  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
