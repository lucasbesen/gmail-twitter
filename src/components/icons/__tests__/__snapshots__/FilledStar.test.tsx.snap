// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render FilledStar icon without error 1`] = `
<svg
  aria-hidden="true"
  className="sc-gzVnrw bdBlBB StyledIconBase-sc-bdy9j4 jbPeIR"
  fill="currentColor"
  focusable="false"
  viewBox="0 0 24 24"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
  />
</svg>
`;
